doc/src/build/devnet.md

```diff
@@ -47,24 +47,35 @@ To connect the Sui client to Sui Devnet, run the following command:
 ```shell
 sui client
 ```
+
 The first time you start Sui client, it displays the following message:
 
-```shell
-Config file ["/Users/dir/.sui/sui_config/client.yaml"] doesn't exist, do you want to connect to a Sui RPC server [y/n]?
 ```
+Config file ["/Users/dir/.sui/sui_config/client.yaml"] doesn't exist, do you want to connect to a Sui full node server [y/n]?
+```
+
 Press **y** and then press **Enter**. It then asks for the RPC server URL: 
 
-```shell
-Sui RPC server Url (Default to Sui Devnet if not specified) :
 ```
-Press **Enter** to connect to Sui Devnet. To use a custom RPC server, enter the URL to the RPC endpoint to use.
+Sui full node server url (Default to Sui Devnet if not specified) :
```

Comment:
```suggestion
Sui Full node server URL (default to Sui Devnet if not specified) :
```

Reply:
I used the exact text displayed to users